scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -376,19 +375,6 @@
        (shutdown-rebalancer))))
 
 (comment
-  (let [mesos-master "zk://sf-test-zk1.pit.twosigma.com:2181,sf-test-zk2.pit.twosigma.com:2181,sf-test-zk3.pit.twosigma.com:2181/mesos"
```

Comment:
Can you leave this here? I think it might be the only place that the rebalancer reconfiguration is demonstrated...at least until we make it easier to configure.